Project and Documentation Cleanup (#12)
K-S-K/ESP32-02-OLedCounter: README.md

#### It was interesting challenge during the project implementation.

> It is happened that it is not a trivial task - to print data to the 128x32 display by the same way we expected after reading about printing to the 128x64 display. Look the [description of the experiment](docs/Ch01_LostBits.rst) if you are interested.

Author: ```suggestion
It was challenging to print data to the 128x32 display in the same way we expected after reading about printing to the 128x64 display. If you are interested, please look at the [description of the experiment](docs/Ch01_LostBits.md).
```
